invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -245,7 +245,7 @@ def community_dashboard_request_view(request, community, community_ui, **kwargs)
     request_is_accepted = request["status"] == AcceptAction.status_to
 
     permissions = community.has_permissions_to(
-        ["update", "read", "search_requests", "search_invites"]
+        ["update", "read", "search_requests", "search_invites", "submit_record"]
```

Comment:
Where does the `submit_record` permission is being consumed?

Reply:
It is used here: https://github.com/inveniosoftware/invenio-communities/pull/1189/files#diff-a4058314680fdd8690322e8b12c653c6b24e629311cc22f6caebc6506cbdea5aR125-R131
Header is being called from `invenio_communities/details/base.html`